examples/multiple_workspaces/README.md

```diff
@@ -0,0 +1,38 @@
+# Multiple workspaces setup
+
+## How to run it:
+
+Try to load the `example.code-workspace` file in VSCode.
+
+You may need to compile the right `.vo` files for the imports to
+work. You can do that with the `Save VO command`; as of now, `coq-lsp`
+will require you do this before opening the depending files.
+
+`coq-lsp` will take care of this automatically in the next version,
+including the auto-update. You can also do:
+
+```sh
+$ coqc -R bar bar bar/barx.y
+$ coqc -R foo foo foo/foox.y
+```
+
+## `coq-lsp` workspace documentation
+
+For each workspace added to a project, `coq-lsp` will try to configure
```

Comment:
```suggestion
One can add multiple folders to a workspace — for instance via the "Add Folder to Workspace" command (or [alternatives](https://code.visualstudio.com/docs/editor/multi-root-workspaces#_adding-folders)).
For each folder added to a workspace, `coq-lsp` will try to configure
```

https://code.visualstudio.com/docs/editor/multi-root-workspaces

Reply:
That is, we should probably stick to the same terms as VsCode docs, and the above is an attempt.

Reply:
Yes, thanks a lot for the suggestions, indeed we need to improve the wording in the line you suggested.